scripts/check_if_for_format.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+REPO_ROOT="$(dirname "$0")"/..
+
+(
+cd $REPO_ROOT
+FORMATERROR=$(git grep -nIE "\<if\(" -- '*.h' '*.cpp')
+FORMATERROR=$FORMATERROR"\n"$(git grep -nIE "\<for\(" -- '*.h' '*.cpp')
+FORMATERROR=$FORMATERROR"\n"$(git grep -nIE "\<if\>\s*\(.*\)\s*\{\s*$" -- '*.h' '*.cpp')
+FORMATERROR=$FORMATERROR"\n"$(git grep -nIE "\<for\>\s*\(.*\)\s*\{\s*$" -- '*.h' '*.cpp')
+
+FORMATERROR=$(echo -e "$FORMATERROR" | grep -v "^[a-zA-Z\./]*:[0-9]*:\s*\/\/")
+FORMATERROR=$(echo -e "$FORMATERROR" | grep -v "^[a-zA-Z\./]*:[0-9]*:\s*\/\*")
+FORMATERROR=$(echo -e "$FORMATERROR" | grep -v "libdevcore")
```

Comment:
Why do you exclude this?

Reply:
the first two are comments, I think it is reasonable to ignore comments

shall we modify the code in libdevcore?
will we need to sync with upstream somehow?

Reply:
we own libdevcore, there is no upstream